blocks/hero-block/features/hero/default.jsx

```diff
@@ -1,30 +1,38 @@
 import React, { Fragment } from "react";
 import PropTypes from "@arc-fusion/prop-types";
+import { RESIZER_TOKEN_VERSION, RESIZER_URL } from "fusion:environment";
+import { useFusionContext } from "fusion:context";
+import { useContent } from "fusion:content";
+import getProperties from "fusion:properties";
 
 // Arc Themes Components - Base set of components used to compose blocks
 // https://github.com/WPMedia/arc-themes-components/
 import {
 	Button,
 	Heading,
 	HeadingSection,
+	imageANSToImageSrc,
 	Paragraph,
 	Picture,
 	Stack,
 } from "@wpmedia/arc-themes-components";
 
 const BLOCK_CLASS_NAME = "b-hero";
 
-const DESKTOP_PHOTO_BREAKPOINT = "(min-width: 700px)";
-
 function Hero({ customFields }) {
 	const {
 		layout = "overlay",
 		alignment = "center",
 		description,
 		headline,
-		imageURLDesktop,
-		imageURLMobile,
-		imageAltText,
+		imageDesktopURL,
+		imageId: desktopImgId,
+		imageDesktopAuth,
+		imageMobileId: imgMobileId,
+		imageMobileAlt,
+		imageMobileAuth,
+		imageMobileURL,
+		resizerURL,
```

Comment:
This will need to be removed, `resizerURL` can be retrieved from `getProperties(arcSite)` like `fallbackImage` is.

Reply:
Fixed this.